src/pages/Usage/Usage.js

```diff
@@ -1,13 +1,14 @@
 import React from "react";
 import Layout from "components/Layout/Layout";
+import Notification from "@canonical/react-components/dist/components/Notification/Notification";
 
 export default function Usage() {
   return (
     <Layout>
-      <div className="p-strip">
-        <div className="row">
-          <h2>Usage</h2>
-        </div>
+      <div className="row">
```

Comment:
Kind of weird to make these changes to a view we can't access :-)

Reply:
It's still accessible via URL.